### ISSUE TITLE
Params to allow classes for govuk-form-group

### ISSUE THREAD

**Issue by web-bert** [alphagov/govuk-frontend#1042]
When using the `govukSelect` macro it would be nice to have a param for adding `classes` to the `govuk-form-group`

I am working on a design where we want to add a background colour to the select and label, rather than having to add a wrapping element or use a generic selector, it would be good to be able to target the form group specifically.

I would  suggest:

```
  {{ govukSelect({
    group: {
      classes: 'my-custom-class'
    }
  }) }}
```

I am not sure if there are other macros that could use the same treatment...

**Comment by igloosi** [comment]
Hi @web-bert, thanks for raising this. A while ago we had a [similar issue](https://github.com/alphagov/govuk-frontend/issues/713) open, but closed it due to lack of interest. what you're describing had been mentioned, so we're going to discuss this as a team early next week. 

We will update the this issue to let you know of the outcome.

**Comment by igloosi** [comment]
Hi @web-bert,
we've deiced to go with a similar syntax as you outlined above in #1043 

We opted for 
```
formGroup: { 
   classes: ''"
} 
```

I'd imagine Select, file upload, input, textarea and date input would be candidates as they all contain the form group wrapper.

You can have a look in the linked radios and checkboxes work to see how we now declare macro params(options). I don't think these examples need to be in the review app or readme files.

I'd suggest a single PR per component for easier review, but also opened to a single PR with meaningful commits to it's easier to follow.

**Comment by igloosi** [comment]
addressed by #1059
